dsc_lib/src/discovery/command_discovery.rs

```diff
@@ -50,7 +51,9 @@ impl CommandDiscovery {
                     let path = entry.path();
                     if path.is_file() {
                         let file_name = path.file_name().unwrap().to_str().unwrap();
-                        if file_name.to_lowercase().ends_with(".dsc.resource.json") {
+                        if file_name.to_lowercase().ends_with(".dsc.resource.json") |
```

Comment:
since we use the lowercase version multiple times, we should probably do it once and store in a variable